app/controllers/courses_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class CoursesController < ApplicationController
+  def new
+    @course = Course.new
+  end
+
+  def edit
+    @course = Course.find(params[:id])
+  end
+
+  def index
+    @courses = Course.all
+  end
+
+  def show
+    @course = Course.find(params[:id])
+    @users = @course.users
+  end
+
+  def create
+    @course = Course.new(course_params)
+    if @course.save
+      flash[:success] = 'course saved!'
+      redirect_to edit_tutor_path(current_user.id)
+    else
+      flash[:alert] = 'course not saved!'
+    end
+  end
+
+  def delete; end
+
+  def destroy; end
+
```

Comment:
Why is there no destroy method?

Reply:
afaik, we didn't have plans to reset any of the tutor class data because 1: the client didn't request that, and 2: they could remove tutors from the site if they no longer tutor and they don't want to show up in searches. Also, I don't see a reason to remove a course from the db unless there is a typo, in which case they could edit it